src/rawdog/llm_client.py

```diff
@@ -46,7 +46,14 @@ def add_message(self, role: str, content: str):
 
     def get_python_package(self, import_name: str):
         base_url = self.config.get("llm_base_url")
-        model = self.config.get("llm_model")
+        model = self.config.get("pip_model")
+        llm_model = self.config.get("llm_model")
+        if model is None:
+            if "ft:" in llm_model or "rawdog" in llm_model or "abante" in llm_model:
+                model = "gpt-3.5-turbo"
+            else:
+                model = llm_model
+
         custom_llm_provider = self.config.get("llm_custom_provider")
```

Comment:
What if the user is using a different llm provider (like ollama)? It would be kind of annoying to have to change the pip model as well. But I guess that's a very niche scenario so it probably doesn't matter.

Reply:
It should only be an issue if they are using ollama with a rawdog fine tuned model. Otherwise they'll just get the same model for both llm_model and pip_model which is what they would want.